js/models/conversations.js

```diff
@@ -2420,14 +2461,23 @@
       });
     },
 
-    async deletePublicMessage(message) {
+    async deletePublicMessages(messages) {
       const channelAPI = await this.getPublicSendData();
       if (!channelAPI) {
         return false;
       }
-      const success = await channelAPI.deleteMessage(message.getServerId());
+
+      let success;
+      if (messages.length > 1) {
+        success = await channelAPI.deleteMessages(
+          messages.map(m => m.getServerId())
+        );
+      } else {
+        success = await channelAPI.deleteMessages([messages[0].getServerId()]);
```

Comment:
looks to me like the if/else is not necessary, unless I'm mistaken?

Reply:
Updated this part